foods/views.py

```diff
@@ -0,0 +1,59 @@
+from django.http import Http404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.parsers import MultiPartParser, FormParser
+
+from .serializers import FoodSerializer, PostFoodSerializer
+from .models import Food, Category
+
+
+class FoodList(APIView):
+    serializer_class = FoodSerializer
+
+    def get(self, request, format=None):
+        foods = Food.objects.all()
+        serializer = self.serializer_class(foods, many=True)
+        return Response(serializer.data)
+
+
+class CreateFood(APIView):
+    parser_classes = (MultiPartParser, FormParser)
+    serializer_class = PostFoodSerializer
+
+    def post(self, request, format='jpg'):
+        files = request.FILES.getlist('files')
+        serializer = self.serializer_class(data=request.data)
+        if serializer.is_valid():
+            serializer.files = files
+            serializer.save()
+            content = {'success': 'Create successful food.'}
+            return Response(content, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class FoodDetail(APIView):
+    serializer_class = FoodSerializer
+    parser_classes = (MultiPartParser, FormParser)
+
+    def get_object(self, pk):
+        try:
+            return Food.objects.get(id=pk)
+        except Food.DoesNotExist:
+            raise Http404
+
+    def get(self, request, pk, format=None):
+        food = self.get_object(pk)
+        serializer = self.serializer_class(food)
+        return Response(serializer.data)
+
+    def put(self, request, pk, format=None):
+        food = self.get_object(pk)
+        files = request.FILES.getlist('files')
+        serializer = PostFoodSerializer(food, data=request.data)
+        if serializer.is_valid():
+            serializer.files = files
+            serializer.update()
+            content = {'success': 'Change successful food.'}
+            return Response(content)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Don't have permission and authetication ??

Reply:
I'll add permission and authentication in another task, is waiting it ..